keyboards/handwired/108keyTrackpoint/rules.mk

```diff
@@ -0,0 +1,22 @@
+MCU = at90usb1286
+F_CPU = 16000000
+ARCH = AVR8
+F_USB = $(F_CPU)
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+OPT_DEFS += -DBOOTLOADER_SIZE=4096
```

Comment:
If the controllers bootloader is known, it should be defined, which will set this to an appropriate value automatically.

https://github.com/qmk/qmk_firmware/blob/504bf11769f1340711d749e8dc818f39a56a4dfb/quantum/template/avr/rules.mk#L42-L59